modules/virt-about-instance-types.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/creating_vms_rh/virt-creating-vms-from-instance-types.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="virt-about-instance-types_{context}"]
+= About instance types
+
+An instance type is a reusable object where you can define resources and characteristics to apply to new VMs. You can define custom instance types or use the variety that are included when you install {VirtProductName}. You can create instance types by using YAML files or the `virtctl` CLI tool.
+
+{VirtProductName} provides two CRDs for configuring instance types: 
+
+* A namespaced object: `VirtualMachineInstancetype`
+* A cluster-wide object: `VirtualMachineClusterInstancetype`
+
+These objects use the same `VirtualMachineInstancetypeSpec`.
+
+[id="required-attributes_{context}"]
+== Required attributes
+
+When you configure an instance type, you must define the `cpu` and `memory` attributes. Other attributes are optional. 
+
+[NOTE]
+====
+When you create a VM from an instance type, you cannot override any parameters defined in the instance type. 
+
+Because CPU and memory resources must be defined in an instance type, requests for these resources are always rejected.
```

Comment:
This seems to be missing something. Not sure if this is any better (or even accurate).

```suggestion
Because CPU and memory resources must be defined in an instance type, requests for these resources  in other objects are always rejected.
```

Reply:
Good point about that line. I will ponder how best to clarify this